recipes/libsndfile/1.0.29/conanfile.py

```diff
@@ -0,0 +1,116 @@
+from conans import ConanFile, CMake, tools
+import os
+
+
+class LibsndfileConan(ConanFile):
+    name = "libsndfile"
+    license = "LGPL-2.1"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "http://www.mega-nerd.com/libsndfile"
+    description = "Libsndfile is a library of C routines for reading and writing files containing sampled audio data."
+    topics = ("audio")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "programs": [True, False],
+        "experimental": [True, False],
+        "with_alsa": [True, False],
+        "with_sqlite": [True, False],
+        "with_external_libs": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "programs": True,
+        "experimental": False,
+
+        "with_alsa": False,
+        "with_sqlite": True,
+        "with_external_libs": True,
+    }
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        if self.options.get_safe("with_alsa"):
+            self.requires("libalsa/1.2.2")
+        if self.options.with_sqlite:
+            self.requires("sqlite3/3.32.3")
+        if self.options.with_external_libs:
+            self.requires("flac/1.3.3")
+            self.requires("ogg/1.3.4")
+            self.requires("opus/1.3.1")
+            self.requires("vorbis/1.3.7")
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+            del self.options.with_alsa
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
```

Comment:
```suggestion
            del self.options.fPIC
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd

```

I didn't build to check, but I'm almost it's a C package. The cmake lists C99: https://github.com/erikd/libsndfile/blob/v1.0.29/CMakeLists.txt#L9. The cpp files are listed for example and test only